pkg/errors/errors.go

```diff
@@ -17,12 +17,15 @@ package errors
 import (
 	"fmt"
 	"strings"
+	"unicode"
+	"unicode/utf8"
 )
 
-func newInvalidCRD(apiGroup string) *KNError {
+func NewInvalidCRD(apiGroup string) *KNError {
 	parts := strings.Split(apiGroup, ".")
 	name := parts[0]
-	msg := fmt.Sprintf("no Knative %s API found on the backend, please verify the installation", name)
+	msg := fmt.Sprintf("no Knative %s API found on the backend, please verify the installation or "+
+		"update the 'kn' client to latest version", firstCharToUpper(name))
```

Comment:
Might be better to call it `matching version`?

Reply:
I wonder whether 'no Knative API' found is appropriate as there could be a Knative API available but too new. So maybe rephrase to "no or newer Knative %s API found on the backend, please verify the installation or update this client"

I just would say update, as it's kind of clear that it should be moved to a matching client.

Reply:
@rhuss addressed in latest update.